pkg/hostagent/pods.go

```diff
@@ -409,6 +417,26 @@ func (agent *HostAgent) creatNullMacEp() {
 
 }
 
+func (agent *HostAgent) creatVethHostEp() {
```

Comment:
Would be better add this check agent.config.OpflexMode == "overlay" and exit. That way, this will execute only if the mode is overlay, even if some files are present.